.github/workflows/Tests.yaml

```diff
@@ -433,3 +433,56 @@ jobs:
           # We get occasional random timeouts, repeat tests to see if
           # it is a random timeout or systematic
           ctest -j2 --output-on-failure -L inputfiles --repeat after-timeout:3
+      - name: Test formaline tar can be built
```

Comment:
1. I'm not strongly opposed to this, but I don't see the benefit of testing formaline only on some configs. What's your reason?

2. I'd rather not build the regular tests a second time. This would double our CI time and it's not clear there's really any benefit. While I agree in that if we had super fast CI it would be best to re-test everything, the main goal of formaline is to be able to reproduce the executable that generated the output and so that's all I want to verify.

Reply:
1. Just to save time. I'm happy to test formaline on all configurations if that doesn't slow CI down too much.
2. Sorry I didn't mean build the tests again. I meant, if you end up testing formaline on a subset of the configurations, you can set `cmake -D USE_FORMALINE=(ON|OFF)` accordingly when building the unit tests.

Reply:
1. ah okay. Once it actually runs through to completion we'll have some estimate of the time and can go from there. Does that sound reasonable?
2. Ah I see. I think by default we just enable it and only disable it on macOS. Are you thinking we can test that running with formaline disabled works correctly?

Reply:
Looks like the test takes 6-7 minutes. I'm on the fence. How dependent do you think formaline is on the compiler configuration? Is it worth the time or would a few sample configurations suffice?

Reply:
Well it happens at the last compile and link stage, so it's extremely dependent on the linker. Less so on the compiler. I would say we could reasonably get away with checking it only on debug builds but I do think we should check all compilers since this is a somewhat obscure feature of linkers/compilers that we're using